src/Command/FixMediaContextCommand.php

```diff
@@ -65,7 +65,7 @@ public function execute(InputInterface $input, OutputInterface $output): int
         }
 
         foreach ($this->mediaPool->getContexts() as $context => $contextAttrs) {
-            /** @var ContextInterface $defaultContext */
+            /** @var ContextInterface|null $defaultContext */
```

Comment:
Instead of phpdoc, a PR should be done to https://github.com/sonata-project/SonataClassificationBundle/blob/3.x/src/Model/ContextManagerInterface.php#L19 in order to extend the template https://github.com/sonata-project/sonata-doctrine-extensions/blob/1.x/src/Model/ManagerInterface.php#L23

Reply:
Removed this line

Reply:
The PR of Classification is now merged.